src/common/server.ts

```diff
@@ -277,6 +255,129 @@ async function createServer(
   return new LanguageClient(serverId, serverName, serverOptions, clientOptions);
 }
 
+async function showWarningMessageWithLogs(message: string, outputChannel: LogOutputChannel) {
+  await vscode.window.showWarningMessage(message, "Show Logs").then((selection) => {
+    if (selection) {
+      outputChannel.show();
+    }
+  });
```

Comment:
Can't this be written as
```suggestion
  const selection = await vscode.window.showWarningMessage(message, "Show Logs");
  if (selection) {
    outputChannel.show();
  }
```